tests/test_wave_03.py

```diff
@@ -55,12 +53,20 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    # Act
+   
+    #Assert
+
+    
+
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
+    assert amandas_data["watched"][0] not in friends_unique_movies
+    assert amandas_data["watched"][1] not in friends_unique_movies
+    assert amandas_data["watched"][2] not in friends_unique_movies
 
-@pytest.mark.skip()
+     
```

Comment:
You've asserted that amanda's movies aren't in `friends_unique_watched`. 

To make the test more robust, you could also check that the movies that the friends have watched are not duplicated and haven't been removed: 

```python
assert INTRIGUE_3 in friends_unique_movies
assert HORROR_1 in friends_unique_movies
assert FANTASY_4 in friends_unique_movies